packages/web-components/src/tree/tree.template.ts

```diff
@@ -0,0 +1,19 @@
+import { children, elements, html } from '@microsoft/fast-element';
+import type { Tree } from './tree.js';
+
+export const template = html<Tree>`
+  <template
+    tabindex="0"
+    @click="${(x, c) => x.handleClick(c.event)}"
+    @focusin="${(x, c) => x.handleFocus(c.event as FocusEvent)}"
+    @focusout="${(x, c) => x.handleBlur(c.event as FocusEvent)}"
+    @keydown="${(x, c) => x.handleKeyDown(c.event as KeyboardEvent)}"
+    @change="${(x, c) => x.handleChange(c.event)}"
```

Comment:
These methods should be named as `<event>Handler`. This allows the entire event name to be present and lowercased, which makes it easier to find. It also prevents collisions with `handleChange`, which is used for FAST's `Notifier.subscribe()` convention (`handleChange` for subscribing, `changeHandler` for `change` events).

* `handleClick` -> `clickHandler`
* `handleFocus` -> `focusHandler`
* `handleBlur` -> `blurHandler`
* `handleKeyDown` -> `keydownHandler`
* `handleChange` -> `changeHandler`